Numerics/EvaluatingFunctions/remez.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This python script implements Remez' algorithm.
+# To use the script, you can modify what happens when this script is executed:
+# See the `if __name__ == "__main__":` branch near the end of this file.
+#
+# Alternatively, `runRemez` can be imported and run directly, see the docstring
+# of runRemez.
+
+import math
+import numpy as np
+from matplotlib import pyplot as plt
+
+# Return n chebyshev nodes on the interval (a,b)
+def _getChebyshevNodes(n, a, b):
+    nodes = [.5*(a+b) + .5*(b-a)*math.cos((2*k+1)/(2.*n)*math.pi) for k in range(n)]
```

Comment:
Since you have NumPy imported already, t his could be easier to read using NumPy broadcasting rather than as a list comprehension.